ios/Classes/NativeTagTechnologyProxy.m

```diff
@@ -0,0 +1,45 @@
+/**
+ * Axway Titanium
+ * Copyright (c) 2009-present by Axway Appcelerator. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+
+#import "NativeTagTechnologyProxy.h"
+#import "TiBlob.h"
+#import "TiNFCTagProxy.h"
+#import "TiNfcUtilities.h"
+#import "TiUtils.h"
+
+@implementation NativeTagTechnology
+
+#pragma mark Public API's
+
+- (void)connect:(id)unused
+{
+  [session connectToTag:tag
+      completionHandler:^(NSError *_Nullable error) {
+        if (error != nil) {
+          [self fireEvent:@"didDetectTags"
```

Comment:
@anil-shukla-axway , I think it is connect event so you should call connect tag event here

Reply:
didDetectTags events is the events where we connect with the tag. after beginning the session, when any tag got found, the didDetectTags event get called.

Reply:
Updated the code.